.github/workflows/e2e-stress-test.yml

```diff
@@ -0,0 +1,465 @@
+name: E2E Stress Test
+
+on:
+  schedule:
+    - cron: '0 12 * * 1-5'
+
+jobs:
+  build:
+    name: Build
+    runs-on: self-hosted
+    outputs:
+      entry_names: ${{ steps.get-changed-apps.outputs.entry_names }}
+      continuous_deployment: ${{ steps.get-changed-apps.outputs.continuous_deployment }}
+
+    env:
+      NODE_EXTRA_CA_CERTS: /etc/ssl/certs/VA-Internal-S2-RCA1-v1.cer.pem
+
+    strategy:
+      fail-fast: false
+      matrix:
+        buildtype: [vagovdev, vagovstaging, vagovprod]
```

Comment:
@theodur , can you please tell me what this should look like? I searched the `workflows` directory for examples but couldn't find any.

I changed it to this for now:
```
      matrix:
        buildtype: vagovprod
```

Reply:
This is done in the [build job](https://github.com/department-of-veterans-affairs/vets-website/blob/6f870139b667d39ce513c1b3c2e6f96af19eda5c/.github/workflows/e2e-tests.yml#L69) of the E2E Tests workflow. The matrix is removed and the build type is referenced as an environment variable

Reply:
Ah, thanks. Like this? https://github.com/department-of-veterans-affairs/vets-website/blob/9a37d51279595d45b0dc78251bd1e6ebea789011/.github/workflows/e2e-stress-test.yml#L8

Reply:
It doesn't look like this is working. The build has failed 2x - https://github.com/department-of-veterans-affairs/vets-website/actions/runs/3129323528/jobs/5078330420

Reply:
Yup, just like that. It looks like it's failing because the matrix references need to be updated to use the environment variable. The 'entry' option in the build command can also be removed